### PR TITLE
[Docs] Fix search plugin

### LINKED ISSUE #1838: [Docs] Search feature is broken
**Description**

Fix the search feature in Meshery docs

---
[Optional] **To Reproduce**
1. Go to 'https://meshery.layer5.io/docs/'
2. Try using the search bar

### PR DESCRIPTION
**Pull request by chandrashritii**
Signed-off-by: Shriti <shritichan@gmail.com>

**Description**

This PR fixes #1838 

- Fixed the search plugin
- Removed the search bar in the top navigation bar
- The site can now include a tags index

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by kushthedude — CHANGES_REQUESTED** [review]
You have already committed `jquery-3.3.1.min.js` then why are you also adding the complete standard jquery `jquery-3.3.1.js`?

**Comment by chandrashritii** [comment]
> 
> 
> You have already committed `jquery-3.3.1.min.js` then why are you also adding the complete standard jquery `jquery-3.3.1.js`?

The jquery files were added by an upstream from the original docsy-jekyll template. Thanks for pointing it out. I'll remove `jquery-3.3.1.js`

**Review by akashgp09 — COMMENTED** [review]
LGTM 👍

File: docs/_config.yml [hunk 1]
```diff
@@ -76,16 +74,6 @@ collections:
   docs:
     output: true
     permalink: /:collection/:path
-  adapters:
```
Comment (leecalcote) [inline docs/_config.yml:79]: These collections need to be included in the config.

File: docs/assets/js/main.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ layout: null
                         return
                     }
                     var query = $(this).val();
-                    var searchPage = "{{ site.url }}/search/?q=" + query;
+                    var searchPage = "/docs/search/?q=" + query;
```
Comment (leecalcote) [inline docs/assets/js/main.js:52]: `baseurl` can be used instead.

File: docs/pages/project/contributing.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ permalink: project/contributing
 ---
 # <a name="contributing">Contributing Overview</a>
 
-Please do! Thanks for your help! :balloon:
+Please do! Thanks for your help!
```
Comment (leecalcote) [inline docs/pages/project/contributing.md:8]: Please revert this change.

File: docs/pages/tags.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (leecalcote) [inline docs/pages/tags.html:1]: Is there an example of how these tags are to be used?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/_config.yml (hunk 1 above)
Comment (leecalcote) [reply docs/_config.yml:79]: Reason being that `sort` (and maybe other?) operations need a collection defined in order to iterate over the set of files.

File: docs/pages/tags.html (hunk 4 above)
Comment (chandrashritii) [reply docs/pages/tags.html:1]: We're not using it at the moment. If we decide to use it, it'll simply provide an index for all the tags defined on the site.

Comment (leecalcote) [reply docs/pages/tags.html:1]: Alrighty.

**Review by leecalcote — APPROVED** [review]
Wonderful, @chandrashritii 👍
